src/test/java/bio/terra/service/search/SearchServiceTest.java

```diff
@@ -0,0 +1,148 @@
+package bio.terra.service.search;
+
+import bio.terra.common.category.Unit;
+import bio.terra.model.SearchIndexRequest;
+import bio.terra.model.SearchQueryRequest;
+import bio.terra.model.SearchQueryResultModel;
+import bio.terra.service.resourcemanagement.google.GoogleProjectResource;
+import bio.terra.service.snapshot.Snapshot;
+import bio.terra.service.snapshot.SnapshotTable;
+import bio.terra.service.tabulardata.google.BigQueryPdao;
+import org.elasticsearch.action.admin.indices.alias.get.GetAliasesRequest;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.GetAliasesResponse;
+import org.elasticsearch.client.IndicesClient;
+import org.elasticsearch.client.RequestOptions;
+import org.elasticsearch.client.RestHighLevelClient;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.search.SearchHits;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.mockito.Mockito.when;
+
+@Category(Unit.class)
+public class SearchServiceTest {
+    private static final String sqlQuery = "SELECT GENERATE_UUID() uuid, CURRENT_TIMESTAMP() as now FROM UNNEST" +
+        "(GENERATE_ARRAY(1, 3));";
+
+    @Mock
+    private BigQueryPdao bigQueryPdao;
+
+    @Mock
+    private RestHighLevelClient client;
+
+    @Mock
+    private IndicesClient indicesClient;
+
+    private SearchService service;
+
+    private SearchIndexRequest searchIndexRequest;
+    private Snapshot snapshot;
+    private List<Map<String, Object>> values;
+
+    @Before
+    public void setup() throws Exception {
+        MockitoAnnotations.openMocks(this);
+        service = new SearchService(bigQueryPdao, client);
+
+        searchIndexRequest = getSearchIndexRequest();
+        snapshot = getSnapshot();
+        values = getSnapshotTableData();
+
+        when(client.indices()).thenReturn(indicesClient);
+    }
+
+    @Test
+    public void indexSnapshotTest() throws Exception {
+        mockGetSnapshotTableData();
+        //service.indexSnapshot(snapshot, searchIndexRequest);
+    }
+
+    @Test
+    public void querySnapshotTest() throws Exception {
+        GetAliasesResponse mockResponse = Mockito.mock(GetAliasesResponse.class);
+        when(mockResponse.getAliases()).thenReturn(Map.of("0f14d0ab-9605-4a62-a9e4-5ed26688389b", new HashSet<>()));
+        when(indicesClient.getAlias(Mockito.any(GetAliasesRequest.class), Mockito.any(RequestOptions.class)))
```

Comment:
I think you can use the no-arg `any()` here, which can infer the class type based on context.

Reply:
In my experience it's better to be explicit about the code where you can, I've been discouraged from using any() when possible in the past

Reply:
In the case of matching a method call argument, I don't see the benefit of passing in the parameter type. If you're reading the test code, does having it provide information that makes it easier to understand what the code is doing?

When the method is overloaded the return type of `any()` is ambiguous and must be provided. But I think it's ok to omit it in other cases.